### ISSUE TITLE
JSON export/import

### ISSUE THREAD

**Issue by FluffyDiscord** [nptr/msstyleEditor#77]
**Is your feature request related to a problem? Please describe.**
Nope

**Describe the solution you'd like**
Option to export and import everything to/from a json and image files.

**Describe alternatives you've considered**
None

**Additional context**
With this feature anyone can make powerful GUIs and also multiple people can work on the same theme as merging would become a thing.

**Comment by nptr** [comment]
I agree that import & export is a necessary feature. It is somewhere on my list, although no due date.
As textual format is intriguing as well. I can probably replace & repurpose the style info export for one half.
